common/AcronymApostrophe.yml

```
@@ -0,0 +1,6 @@
+extends: existence
+message: "The acronym [%s] should not be in possesive form or used with an apostrophe in plural form."
+level: error
+ignorecase: true
+tokens:
+  - ^[A-Z]{2,}'(?:s|es|ies)
```

```suggestion
  - ^[A-Z]{2,}'(?:s|es|ies)?
```

Maybe this to catch:
- invalid apostrophe: `USA' army`
- acronym ending with S, that may be used like this: `ISS' members`

---

Good idea, I've added this functionality too.